labonneboite/common/util.py

```diff
@@ -66,30 +66,25 @@ def user_is_pro():
     - Office data : statistics about recruitments...
     """
 
+    # Check user IP (not need to be authenticated)
     user_ip = get_user_ip()
+    if user_ip in settings.VERSION_PRO_ALLOWED_IPS:
+        return True
 
-    # =============================================================
-    # beginning of temporary block of version pro (FIXME remove me)
-    if current_user.is_authenticated:
-        current_user_email = current_user.email.lower()
-        result = (current_user_email in settings.VERSION_PRO_ALLOWED_EMAILS)
-        return result
-    else:
-        return False
-    # end of temporary block of version pro (FIXME remove me)
-    # =============================================================
-
+    # Check user e-mail by plain_value, suffix or regex (@see local_settings.py)
     if current_user.is_authenticated:
         current_user_email = current_user.email.lower()
 
-        result = (user_ip in settings.VERSION_PRO_ALLOWED_IPS
-            or current_user_email in settings.VERSION_PRO_ALLOWED_EMAILS
+        result = (current_user_email in settings.VERSION_PRO_ALLOWED_EMAILS
             or any(current_user_email.endswith(suffix) for suffix in settings.VERSION_PRO_ALLOWED_EMAIL_SUFFIXES)
             or any(re.match(regexp, current_user_email) is not None for regexp in settings.VERSION_PRO_ALLOWED_EMAIL_REGEXPS)
             )
 
     return result
```

Comment:
@AlexandreCantin  bug vu en deploy prod

Traceback (most recent call last):
  File "/home/deploy/config/labonneboite/local/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/deploy/config/labonneboite/local/lib/python2.7/site-packages/werkzeug/contrib/fixers.py", line 148, in __call__
    return self.app(environ, start_response)
  File "/home/deploy/config/labonneboite/local/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/home/deploy/config/labonneboite/local/lib/python2.7/site-packages/flask/app.py", line 1547, in handle_exception
    return self.finalize_request(handler(e), from_error_handler=True)
  File "/home/deploy/code/current/labonneboite/labonneboite/web/app.py", line 359, in internal_error
    return render_template('error/500.html'), 500
  File "/home/deploy/config/labonneboite/local/lib/python2.7/site-packages/flask/templating.py", line 132, in render_template
    ctx.app.update_template_context(context)
  File "/home/deploy/config/labonneboite/local/lib/python2.7/site-packages/flask/app.py", line 764, in update_template_context
    context.update(func())
  File "/home/deploy/code/current/labonneboite/labonneboite/web/app.py", line 183, in inject_dict_for_all_templates
    'hotjar_tag': util.get_hotjar_tag(),
  File "/home/deploy/code/deploys/2017_09_22_15_12_16/labonneboite/common/util.py", line 98, in get_hotjar_tag
    if user_is_pro():
  File "/home/deploy/code/deploys/2017_09_22_15_12_16/labonneboite/common/util.py", line 83, in user_is_pro
    return result
UnboundLocalError: local variable 'result' referenced before assignment